github-sample/workflows/build-publish.yml

```diff
@@ -301,6 +270,42 @@ jobs:
       run: |
         echo $docker_digest
         echo "digest=$docker_digest" >> $GITHUB_OUTPUT
+  docker-build-auth-predeployment:
```

Comment:
Is there a reason we are duplicating service build with a pre-deployment build? Is the original service build going to be deprecated later?

Reply:
It's another service, so it needs another image